### ISSUE TITLE
CORS warnings

### ISSUE THREAD

**Issue by jpzwarte** [chromaui/storybook-addon-pseudo-states#1]
```
Can't access cssRules, most likely due to CORS restrictions https://use.typekit.net/zkq0zzv.css
```

Anyway to detect this and prevent "spamming" the console?

**Comment by ghengeveld** [comment]
Yeah I noticed this too when loading webfonts from Google. Let's avoid logging the same warning over and over again.

**Comment by ghengeveld** [comment]
Fixed in 1.0.0-rc.3
